ebuild-maintenance/git/text.xml

```diff
@@ -287,13 +284,13 @@ are optionally used in Gentoo:
     to the commit.
   </li>
   <li>
-    <c>Package-Manager:</c> This is automatically inserted by
-    <c>repoman commit</c> and it specifies the version of
+    <c>Package-Manager:</c> This was automatically inserted by
+    <c>repoman commit</c> and specified the version of
     <c>sys-apps/portage</c> on the system.
   </li>
   <li>
-    <c>RepoMan-Options:</c> This is automatically inserted by
-    <c>repoman commit</c> and records the options passed to repoman (such as
+    <c>RepoMan-Options:</c> This was automatically inserted by
+    <c>repoman commit</c> and recorded the options passed to repoman (such as
```

Comment:
Past tense is somewhat misleading, because it suggests that this functionality was removed from repoman.

Maybe just "Automatically inserted by `repoman` and specifies ..."?

Reply:
Sounds good, thanks!